src/net_processing.cpp

```diff
@@ -2396,6 +2407,14 @@ void PeerManagerImpl::ProcessBlock(CNode& pfrom, const std::shared_ptr<const CBl
 void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type, CDataStream& vRecv,
```

Comment:
do I understand correctly that after these changes `PeerManagerImpl::ProcessMessage` is a test-only function? If so, I think the comment in `net_processing.h` should be updated.

Reply:
`PeerManager::ProcessMessage` has always only been exposed for tests, and is already documented that way in the header file:

```c++
    /** Process a single message from a peer. Public for fuzz testing */
    virtual void ProcessMessage(CNode& pfrom, const std::string& msg_type, CDataStream& vRecv,
```

`PeerManagerImpl::ProcessMessage` is thus needed to complete the `PeerManager` interface.